services/scan/src/store.test.ts

```diff
@@ -578,6 +590,30 @@ test('exportCvrs', () => {
     expect.stringContaining(stateOfHamilton.election.precincts[0].id)
   );
 
+  // Confirm that ballot images and layouts are included when building the CVR
+  expect(buildCastVoteRecordMock).toHaveBeenCalledWith(
```

Comment:
We test the content of the CVR in `build_cast_vote_record.test.ts` so instead of doing that again here, I wrote a test expecting that we call `buildCastVoteRecord` with the correct arguments.